vedro_allure_reporter/_allure_reporter.py

```diff
@@ -44,20 +44,39 @@ def __init__(self, config: Type["AllureReporter"], *,
         self._project_name = config.project_name
         self._report_dir = config.report_dir
         self._attach_scope = config.attach_scope
+        self._allure_labels = config.allure_labels
         self._attach_artifacts = config.attach_artifacts
         self._config_labels = config.labels
 
+    async def on_startup(self, event: StartupEvent) -> None:
+        if self._allure_labels is None:
+            return
+
+        labels = set()
+        for label_str in self._allure_labels:
+            name, value = label_str.split("=")
+            label = (name, value)
+            labels.add(label)
+
+        async for scenario in event.scheduler:
+            scenario_labels = set([(label.name, label.value) for label in self._get_scenario_labels(scenario)])
+            if labels.isdisjoint(scenario_labels):
```

Comment:
I was expecting that specifying multiple labels would be interpreted as an AND condition, not an OR.